samples/nrf_desktop/src/hw_interface/buttons_matrix.c

```diff
@@ -191,17 +294,26 @@ void button_pressed(struct device *gpio_dev, struct gpio_callback *cb,
 		}
 	}
 
-	if (!atomic_get(&active)) {
+	switch (state) {
+	case STATE_IDLE:;
```

Comment:
remove semicolon at the end of line

Reply:
this is intentional to keep event declaration under case
it can be that checkpatch will reject it (once it starts to work again) but if not then I would prefer to keep it

Reply:
I see. What about putting the empty statement in a new line? It will look intentional then.